src/content/datachannel/datatransfer/index.html

```diff
@@ -46,7 +46,7 @@
 
     <section>
       <div>
-        <input type="number" id="megsToSend" min="1" name="megs" value="1048"/>
+        <input type="number" id="megsToSend" min="1" name="megs" value="128"/> Megs
         <button id="sendTheData" type="button">Generate and send data</button>
```

Comment:
Ok, sure. How do I know it actually worked to go into unordered mode? I'm passing {ordered: true/false} into createDataChannel now based on the value of the checkbox.